com.microsoft.mrtk.input/Editor/InputValidation.cs

```diff
@@ -54,5 +58,50 @@ private static BuildValidationRule GenerateSkinWeightsRule()
                 Error = false
             };
         }
+
+        private static BuildValidationRule GenerateGLTFastRule()
+        {
+            return new BuildValidationRule()
+            {
+                Category = "MRTK3",
+                Message = "For controller models to show up in scenes, you need a glTF importer. We recommend the open source glTFast importer. " +
+                "Please ignore this message if you already have another glTF importer in the project.",
+                CheckPredicate = () =>
+                {
+#if GLTFAST_PRESENT
+                    return true;
+#else
+                    return false;
+#endif
+                },
+                FixIt = () => Application.OpenURL("https://github.com/atteneder/glTFast"),
+                FixItMessage = "Open the GitHub repo for glTFast. Please refer to the installing section for instructions.",
+                FixItAutomatic = false,
+                Error = false
+            };
+        }
+
+        private static BuildValidationRule GenerateKTXRule()
+        {
+            return new BuildValidationRule()
+            {
+                Category = "MRTK3",
+                Message = "For Meta's controller models to show up in scenes, you need KTX support for your glTF importer. " +
+                "If you are using our recommended importer glTFast, you need to also include the open source KTX Unity package in the project. " +
+                "Please ignore this message if you already have another glTF importer with KTX support in the project.",
```

Comment:
Do we want to provide a mechanism for "ignore", like we did in MRTK2?

Reply:
Unfortunately if we want to rely on Unity's validator we have to face the reality of them not currently providing this feature. We could hypothetically do something on our side, but that would like to be a new feature we add in a follow-up.